### PR TITLE
Constructable Filing Cabinets

### PR DESCRIPTION
**Pull request by 567Turtle**
Construct filing cabinets for 4 metal sheets

## About The Pull Request

Literally just made it so you can construct a filing cabinet for 2 iron sheets

## Why It's Good For The Game

I constantly find myself going "Damn, if only there was a way I could make filing cabinets!" And now there is. For 2 iron sheets, you can construct a filing cabinet. This was mostly made as a beginning practice for myself as I learn C and try to do some coding.

## Changelog

:cl: 567Turtle
expansion: Filing cabinets can be constructed for 2 iron sheets
/:cl:

### CODE REVIEW

File: code/game/objects/items/stacks/sheets/sheet_types.dm [hunk 1]
```diff
@@ -107,6 +107,7 @@ GLOBAL_LIST_INIT(metal_recipes, list ( \
 	new/datum/stack_recipe("button frame", /obj/item/wallframe/button, 1), \
 	null, \
 	new/datum/stack_recipe("iron door", /obj/structure/mineral_door/iron, 20, one_per_turf = TRUE, on_floor = TRUE), \
+	new /datum/stack_recipe("filing cabinet", /obj/structure/filingcabinet, 15, time = 10 SECONDS, one_per_turf = TRUE, on_floor = TRUE), \
```
Comment (tralezab) [inline code/game/objects/items/stacks/sheets/sheet_types.dm:110]:
```suggestion
	new/datum/stack_recipe("filing cabinet", /obj/structure/filingcabinet, 15, time = 10 SECONDS, one_per_turf = TRUE, on_floor = TRUE), \
```

**Review by tralezab — CHANGES_REQUESTED** [review]
make the price of constructing the cabinet the same as what you get from destroying it, please. otherwise good first pr

**Comment by 567Turtle** [comment]
> make the price of constructing the cabinet the same as what you get from destroying it, please. otherwise good first pr

Did and done, it should be 2.

**Review by tralezab — APPROVED** [review]

**Comment by maxymax13** [comment]
hahaha nukies you fools, i have stolen your pinpointer! if you want to find the disk you will have to go in this room and open every one of these 500 filing cabinets until you find the disk, and there will be food items renamed with a pen to say "nuclear authentication disk"! have fun! wait what are you doing with that rocket launch

**Comment by Mothblocks** [comment]
I constantly find myself going "Damn, if only there was a way I could make filing cabinets!"

**Pull request merged**